structuredevent-service-cdp/src/main/java/com/sequenceiq/cloudbreak/structuredevent/repository/CDPStructuredEventRepository.java

```diff
@@ -72,7 +72,7 @@ default List<ResourceBasicView> findAllResourceBasicViewByNamesAndAccountId(Coll
     }
 
     @Modifying
-    @Query("DELETE FROM CDPStructuredEventEntity cdse WHERE cdse.resourceCrn = :accountId AND cdse.timestamp <= :timestamp")
+    @Query("DELETE FROM CDPStructuredEventEntity cdse WHERE cdse.accountId = :accountId AND cdse.timestamp <= :timestamp")
```

Comment:
How it did not cause any issues so far? Actually, it seems that previously this query affected exactly 0 records...